GitUI/UserControls/FileStatusList.cs

```diff
@@ -332,7 +335,7 @@ public IEnumerable<GitItemStatus> SelectedItems
 
         [DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)]
         [Browsable(false)]
-        public GitItemStatus SelectedItem
+       public GitItemStatus SelectedItem
```

Comment:
Thanks, fixed. Thanks to @vbjay for the tip. In this case I should have just run ctrl+k, ctrl+f, would be enough.